hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -603,4 +603,22 @@ public String toString() {
                 + ", indexStats=" + indexStats
                 + '}';
     }
+
+    /**
+     * Converts the value in nanoseconds to milliseconds representation. The value in range 0-999999ns will be counted as 1 ms.
+     * This is done to differ an empty zero value from a value that less than our current resolution which is 1 ms.
+     * @param nanos the value in nanoseconds that will be converted to milliseconds.
+     * @return value in milliseconds.
+     */
+    private static long convertNanosToMillis(long nanos) {
```

Comment:
`TimeUtil#timeInMsOrOneIfResultIsZero`?

Reply:
Yes, it's identical. Thanks for pointing me out. Fixed.